CONTRIBUTING.md

```diff
@@ -29,23 +27,19 @@ Documentation is kept and published via a git repository:
 
 ## Workflow
 
-### JIRA and Github issues
+### Github issues
 
-The project uses a JIRA and also githb issues to track work.  Please create one
-of a JIRA issue or a github issue so that we can track a contribution.
-
-JIRA:
-
-    https://issues.apache.org/jira/browse/JENA
+The project github issues to track work.  Please create one
+of a github issue so that we can track a contribution.
 
 Github issue:
 
     https://github.com/apache/jena/issues
 
 ### Github
 
-It is useful to create a JIRA then use the JIRA number (e.g. JENA-9999 or GH-9999)
-in the Pull Request title. This activates the automated mirroring of
+It is useful to create a Gihhub issue use the issue number (e.g. GH-9999)
```

Comment:
s/Gihhub/GitHub, and "to create a GitHub issue to use the issue number"? Or something like to use/and use/etc?

Reply:
This can be simplified - we don't send issues discussions to dev@ anymore, only the tracking mailing list.